frontend/public/components/resource-quota.jsx

```diff
@@ -175,11 +175,11 @@ export const ResourceUsageRow = ({ quota, resourceType }) => {
   );
 };
 
-const NoQuotaGuage = ({ title }) => (
-  <GaugeChart error="No Quota" thresholds={[{ value: 100 }]} title={title} />
+const NoQuotaGuage = ({ title, className }) => (
+  <GaugeChart error="No Quota" thresholds={[{ value: 100 }]} title={title} className={className} />
 );
 
-export const QuotaGaugeCharts = ({ quota, resourceTypes }) => {
+export const QuotaGaugeCharts = ({ quota, resourceTypes, chartClassName = null }) => {
```

Comment:
I don't think we need to default to null?

```suggestion
export const QuotaGaugeCharts = ({ quota, resourceTypes, chartClassName }) => {
```

Reply:
that's to tell TS `chartClassName` is optional parameter. Another option would be to use `@type` annotation.